src/context/data.ts

```diff
@@ -1,14 +1,11 @@
 import { ResponseTransformer } from '../response'
+import { json } from './json'
 
 export type DataContext<T> = (payload: T) => ResponseTransformer
 
 /**
  * Returns a GraphQL body payload.
  */
 export const data: DataContext<Record<string, any>> = (payload) => {
-  return (res) => {
-    res.headers.set('Content-Type', 'application/json')
-    res.body = JSON.stringify({ data: payload })
-    return res
-  }
+  return json({ data: payload }, { merge: true })
```

Comment:
Not sure why this wasn't using `json` just as `errors`. A likely worthwhile change regardless.

Reply:
Most likely for historic reasons. I agree with your refactor here: it should reuse the `json` utility.